libgalois/include/galois/graphs/FileGraph.h

```diff
@@ -48,6 +48,15 @@
 #include "galois/substrate/NumaMem.h"
 #include "galois/Reduction.h"
 
+template <typename T>
+struct SizeOf {
+  static constexpr std::size_t value = sizeof(T);
+};
+template <> // sizeof never returns 0
```

Comment:
Why do you need this?

Reply:
As you've seen, it's for determining the size of edge data. See my reply below for more details.